package.json

```diff
@@ -42,8 +42,8 @@
         "glob": "^7.1.6",
         "laravel-vite-plugin": "^0.7.8",
         "postcss": "^8.2.7",
-        "postcss-discard-comments": "^4.0.2",
```

Comment:
Nitpick: Major version update. Do those packages have breaking changes that affect us?

Reply:
I just checked, it's fine as far as I see.